meals.css

```diff
@@ -51,3 +51,6 @@ width: 200px;
     margin-top: 100px;
     margin-bottom: 100px;
 }
+.new-food-btn{
+    margin-bottom: 20px;
```

Comment:
our styling for this will most likely change once we start adding more to this section